packages/math-input/src/components/keypad/keypad-mathquill.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import MathQuill from "mathquill";
+import * as React from "react";
+
+import Key from "../../data/keys";
+import keyTranslator from "../key-translator";
+
+import Keypad from "./index";
+
+export default {
+    title: "v2 Keypad With Mathquill",
+};
+
+const mathQuillConfig = {
+    autoCommands: "pi theta phi sqrt nthroot",
+    charsThatBreakOutOfSupSub: "+-*/=<>≠≤≥",
+    supSubsRequireOperand: true,
+    spaceBehavesLikeTab: true,
+};
+
+export function V2KeypadWithMathquill() {
+    const mathquillWrapperRef = React.useRef<HTMLDivElement>(null);
+    const [mathQuill, setMathQuill] = React.useState<MathQuill>();
+
+    React.useEffect(() => {
+        if (!mathQuill && mathquillWrapperRef.current) {
+            const MQ = MathQuill.getInterface(2);
```

Comment:
What does `2` stand for??

Reply:
I think it's just picking [MathQuill's API interface](https://docs.mathquill.com/en/latest/Api_Methods/#api-methods). According to the docs "2" is the latest and it's also what we're using everywhere.